src/trainer.py

```diff
@@ -27,10 +27,10 @@
 from .utils import config
 
 
-def build_trainer_params(args, task_names):
+def build_trainer_params(args, task_names, phase='pretrain'):
```

Comment:
how come we didn't need to pass this argument in before?

Reply:
because before, we used the same value for both pretraining and target task training stages, so if we want to separate it out we have to pass the phase in